lib/infra_conversion_throttler.rb

```diff
@@ -62,7 +62,7 @@ def self.pending_conversion_jobs
 
   # @return [Hash] the list of jobs in state 'running', grouped by conversion host
   def self.running_conversion_jobs
-    running = InfraConversionJob.where.not(:state => ['waiting_to_start', 'finished'])
+    running = InfraConversionJob.where.not(:state => ['waiting_to_start', 'finished']).reject { |job| job.migration_task&.conversion_host.nil? }
```

Comment:
@fdupont-redhat since this is a case that shouldn't happen I wonder if it'd be better to log an error [here](https://github.com/ManageIQ/manageiq/blob/9085751ba7bae6fb5ad86080719ba5b57017840c/lib/infra_conversion_throttler.rb#L85) with the jobs that don't have a conversion host.